arrows/vtk/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ endif()
 
 target_link_libraries( kwiver_algo_vtk
   PUBLIC               vital
+                       kwiver_algo_pdal
```

Comment:
This should go in the `PRIVATE` section below because the PDAL includes are only used in the `.cxx` file and not exposed in the `.h`.

Reply:
Adding this change this causes the build to fail. Could the build failing have something to do with some detail in `kwiver_algo_pdal`'s [creation](https://github.com/Kitware/kwiver/blob/master/arrows/pdal/CMakeLists.txt#L28)? Or possibly the fact that it is a static library?

Reply:
Can you post the error?

Reply:
```bash
/usr/bin/ld: CMakeFiles/kwiver_algo_vtk_applets.dir/fuse_depth.cxx.o: in function `kwiver::arrows::vtk::fuse_depth::priv::write_isosurface()':
fuse_depth.cxx:(.text._ZN6kwiver6arrows3vtk10fuse_depth4priv16write_isosurfaceEv[_ZN6kwiver6arrows3vtk10fuse_depth4priv16write_isosurfaceEv]+0x48b): undefined reference to `kwiver::arrows::pdal::pointcloud_io::save_(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, kwiver::vital::local_geo_cs const&, std::vector<Eigen::Matrix<double, 3, 1, 0, 3, 1>, std::allocator<Eigen::Matrix<double, 3, 1, 0, 3, 1> > > const&, std::vector<kwiver::vital::rgb_color, std::allocator<kwiver::vital::rgb_color> > const&)'
collect2: error: ld returned 1 exit status
```